partner_shipping_policy/models/res_partner.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class Partner(models.Model):
+    _inherit = "res.partner"
+
+    picking_policy = fields.Selection(
+        [("direct", "As soon as possible"), ("one", "When all products are ready")],
```

Comment:
Perhaps it would be great to get the selection values from the `picking_policy` field from the `sale.order`? Then, if more values are added in custom modules for example, it will already be caught here.